src/sdk/sdk.ts

```diff
@@ -87,9 +93,17 @@ export class FusionSDK {
             throw new Error('quoter has not returned quoteId')
         }
 
+        let nonce = params.nonce
```

Comment:
1. `nonce = params.nonce !== undefined ? params.nonce : OrderNonce.Auto` (Auto by default)
2. Make it in separate function `getOrderNonce(nonce: OrderNonce | number | string, walletAddress: string): number | string | undefined`
3. In case of OrderNonce.Empty - return undefined